missioncontrol/middleware.py

```diff
@@ -0,0 +1,53 @@
+# this functionality cribbed from treeherder:
+# https://github.com/mozilla/treeherder/blob/2501fbc10ad4ec8da6cb4d1a49472f866659ed64/treeherder/middleware.py
+
+import re
+
+from whitenoise.middleware import WhiteNoiseMiddleware
+
+
+class CustomWhiteNoise(WhiteNoiseMiddleware):
+    """
+    Adds two additional features to WhiteNoise:
+
+    1) Serving index pages for directory paths (such as the site root).
```

Comment:
This part of the middleware will thankfully be redundant when WhiteNoise 4 is released (currently in release candidate). When that happens I plan on updating the Treeherder version, which you are welcome to borrow too. For now this should work fine :-)